SAMLPortal/Controllers/HomeController.cs

```diff
@@ -15,11 +15,8 @@ namespace SAMLPortal.Controllers
 	[Authorize]
 	public class HomeController : Controller
 	{
-		private readonly ILogger<HomeController> _logger;
-
-		public HomeController(ILogger<HomeController> logger)
+		public HomeController()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/luca_6/SAMLPortal/pullRequest?prid=5339427)